WcaOnRails/app/mailers/competitions_mailer.rb

```diff
@@ -8,7 +8,7 @@ def notify_board_of_confirmed_competition(confirmer, competition)
     @confirmer = confirmer
     mail(
       to: "board@worldcubeassociation.org",
-      cc: competition.delegates.pluck(:email),
+      cc: competition.delegates.pluck(:email) + competition.delegates.map { |d| d.senior_delegate&.email }.compact.flatten(1),
```

Comment:
1. `flatten' looks weird. Can `senior_delegate&.email` really be an array?

2. But several delegates can probably have the same senior delegate. So `.compact.uniq` is maybe the right way?